lib/dest/index.js

```diff
@@ -12,11 +12,21 @@ function dest(outFolder, opt) {
   }
 
   function saveFile(file, enc, cb) {
+    /* jshint validthis: true */
+    var stream = this;
+
+    function complete(err, file) {
+      cb(err, file);
+      if (!err) {
```

Comment:
@davidchase you are correct that the flag means more data can be pushed; however, that has no bearing on if we can read more.  As soon as we are process the file object and pass it on (calling the callback with err & file), we are able to read another and process that.  As far as I am aware, backpressure is handled by the reading stream and because we weren't reading or resuming, the beginning of our stream was clogging the pipeline.  As you stated, the piping of dest to somewhere else will drain it, so the read will essentially nothing in that case (I think).